### ISSUE TITLE
No longer on the Play Store? 

### ISSUE THREAD

**Issue by calvarez-ov** [tunitowen/DevDrawer#38]
I love this app! I was about to recommend it to some colleagues, and then I saw it wasn't on the Play Store anymore 😞 

Any chance you'll put it back up?

Thanks!

**Comment by calvarez-ov** [comment]
I just discovered another app inspired by yours: https://github.com/tasomaniac/DevWidget
Looks like that app has also been removed from the play store :( I'll open a ticket there too ;)

**Comment by Kisty** [comment]
Is this legit? https://play.google.com/store/apps/details?id=ch.pete.DevDrawer

**Comment by tasomaniac** [comment]
As far as I see they just forked the repo and republished it. Don't think the license (or lack of it) in this repo allows that legally. 

I created an app widget from scratch and I am actively developing it. It is more modern looking and already has more features. Feel free to check it out. 

https://play.google.com/store/apps/details?id=com.tasomaniac.devwidget

**Comment by tunitowen** [comment]
Yeah looks like they have just forked it. The code in this repo is disgusting, I have quite a few times now, started to re-write but just don't have the time.

@tasomaniac the app looks great, best of luck 👍

**Comment by hameno** [comment]
For anyone interested, I've been working on my fork for a while and finally had time to finish it. It started as a fork but is now completely rewritten.
It has multiple unique features compared to the original app: 
_**Filter apps based on their signature**_
This means that you can select all apps that share the same singing key, e.g. all locally developed apps signed with the same debug key.
_**multiple widgets with different configurations**_
You can have multiple widgets that each have their own filtering config.

Public beta on the Play Store is available now: https://github.com/PSDev/DevDrawer
